### DISCUSSION TITLE
Enhancement of the script in pyproject.toml

### DISCUSSION THREAD

**Discussion by Saphyel** [python-poetry/poetry#6294] [Ideas]
I was wondering if it will be possible to change how works the script section because at the moment is limited to run a python script only.

I saw several python projects that have something like this implemented https://pdm.fming.dev/latest/usage/scripts/
so you can execute shell commands, etc... The reality is this is more useful as you can execute commands and chain them in the scripts
Other languages like node, php, java, etc... they have a main/popular dependency manager supporting this so not sure why poetry is more limited even within the python world?

**Comment by radoering** [comment]
There is a plugin: https://github.com/nat-n/poethepoet
